_posts/data-announces-formation.md

```diff
@@ -8,7 +8,8 @@ author:
   picture: "/assets/blog/authors/data-team.png"
 ogImage:
   url: "/assets/blog/data-announces-formation/data-meeting-july-20.webp"
-banner: "[Read our latest press release announcing our formation.]($slug)"
+banner: 
+  text: "Read our latest press release announcing our formation."
```

Comment:
Just for future reference to add a banner would be the following.
 
```
banner: 
  text: "Text in here"
```

but to hide it would be 

```
banner: undefined
```
or
```
banner: _blank
```

Is that correct?

Reply:
Hide it this way:
```
banner:
  hide: true
```